### PR TITLE
Prepare Release 0.8.6

### PR DESCRIPTION
**Pull request by maxlambrecht**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## [0.8.6] - 2024-03-04
+
+### Dependency updates
+
+- Bump `com.google.protobuf:protoc` from 3.25.2 to 3.25.3 (#218)
+- Bump `io.grpc:grpc-protobuf` and  `io.grpc:grpc-stub` from 1.61.1 to 1.62.2 (#222)
```
Comment (rturner3) [inline CHANGELOG.md:8]: I see a few other gRPC and Netty dependency bumps not mentioned here, should we call those out too? https://github.com/spiffe/java-spiffe/compare/v0.8.5...main

Comment (maxlambrecht) [reply CHANGELOG.md:8]: Good catch, I added the missing Netty dependency bump.

Which other gRPC dependencies? I included the two that are imported:  `io.grpc:grpc-protobuf` and  `io.grpc:grpc-stub`.

Comment (rturner3) [reply CHANGELOG.md:8]: The two I noticed were `io.grpc:grpc-netty` and `io.grpc:grpc-netty-shaded`

https://github.com/spiffe/java-spiffe/commit/6e993c4350ee8297bbc8963e64b7019283dbd45b

Comment (maxlambrecht) [reply CHANGELOG.md]: You are right, thanks for catching it!

**Review by rturner3 — APPROVED** [review]

**Pull request merged**